legacy/Dockerfile

```diff
@@ -27,6 +27,10 @@ RUN { \
 		echo 'opcache.enable_cli=1'; \
 	} > /usr/local/etc/php/conf.d/opcache-recommended.ini
 
+# SQLite Directory Setup
+RUN mkdir -p /var/www/data \
```

Comment:
The `php` image that we are extending from sets the webroot as `/var/www/html`
https://github.com/docker-library/php/blob/903540ea7918b5cabed6b32e81f8518f9e6f204f/7.1/apache/Dockerfile
so unless someone explicitly sets it to something else then that's what it will be. I'm not sure why anyone would go to the trouble to override that though without also fixing the permissions. And if the were going to override that, why would they pick a directory just one level higher?